radiation/FatesRadiationDriveMod.F90

```diff
@@ -438,11 +439,18 @@ subroutine FatesSunShadeFracs(nsites, sites,bc_in,bc_out)
                            if_notair: if (ft>0) then
                               area_frac = twostr%scelg(cl,icol)%area
                               vai = twostr%scelg(cl,icol)%sai+twostr%scelg(cl,icol)%lai
-                              nv = minloc(dlower_vai, DIM=1, MASK=(dlower_vai>vai))
+
+                              nv = GetNVegLayers(vai)
+
                               do iv = 1, nv
                                  
-                                 vai_top = dlower_vai(iv)-dinc_vai(iv)
-                                 vai_bot = min(dlower_vai(iv),twostr%scelg(cl,icol)%sai+twostr%scelg(cl,icol)%lai)
+                                 vai_top = dlower_vai(iv)
+
+                                 if(iv == nv) then
+                                    vai_bot = twostr%scelg(cl,icol)%sai+twostr%scelg(cl,icol)%lai
+                                 else
+                                    vai_bot = dlower_vai(iv)+dinc_vai(iv)
```

Comment:
Could this match the change in FatesAllometryMod line 3197: `vai_bot = dlower_vai(iv+1)`

Reply:
yes, that's true!